pages/sdk/relay-kit/guides/onchain-identifier.mdx

```diff
@@ -0,0 +1,199 @@
+import { Tabs, Steps, Callout } from 'nextra/components'
+
+# Track ERC-4337 transactions with an onchain identifier
+
+In this guide, you will learn how to get an identifier and track all your ERC-4337 Safe transactions onchain by attaching it to the user operations `callData`.
+
+This guide summarizes all the steps included in the [Safe accounts with the Safe4337Module guide](./4337-safe-sdk.mdx), with an additional step to add the `onchainIdentifier` to your Safe transactions. If you already have the Relay Kit integrated, continue reading to learn how to get your onchain identifier, and feel free to jump straight to [step number 4](#add-your-onchain-identifier-to-the-user-operation-calldata) to learn how to use it afterward.
+
+Please check the Safe4337Module linked guide for all the details regarding the steps summarized here.
```

Comment:
Could potentially remove this statement as it's similar to the first statement of the last paragraph.